src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -78,10 +82,14 @@ export default class DetailsBody extends React.Component<Props> {
       ({label}) => label === 'warning'
     );
 
+    const dataSource = incident.alertRule?.dataset;
+
     return (
       <RuleDetails>
         <span>{t('Data Source')}</span>
-        <span>{t(toTitleCase(incident.alertRule?.dataset))}</span>
+        <span>
+          {t(toTitleCase(dataSource === Dataset.ERRORS ? 'errors' : dataSource))}
```

Comment:
I would also be OK having a mapping of `Dataset` enums to strings.

This is already actually wrong, since passing dynamic content to `t` doesn't allow the text extractor to translate these strings.